### PR TITLE
Using Sylius template event to change logo

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | 1.10
| License         | MIT

After:
image

@CoderMaggie I added the hint about yaml files, if it does not fit I can remove it 🚀

### CODE REVIEW

File: docs/getting-started-with-sylius/shop-customizations.rst [hunk 1]
```diff
@@ -30,7 +30,24 @@ Next, replace the ``img`` element source with a link to the logo or properly imp
 `Symfony assets documentation <https://symfony.com/doc/current/best_practices/web-assets.html>`_ for more info).
 
 The other way to achieve this is to modify the configuration of the ``sylius.shop.layout.header.grid`` template event.
-Learn more about it in :doc:`this doc</customization/template>`.
+Using the same logo file as from example above ``templates/bundles/SyliusShopBundle/Layout/Header/_logo.html.twig``.
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/shop-customizations.rst:33]:
```suggestion
Using the same logo file as from the example above ``templates/shop/Layout/Header/_logo.html.twig``.
```

File: docs/getting-started-with-sylius/shop-customizations.rst [hunk 2]
```diff
@@ -30,7 +30,24 @@ Next, replace the ``img`` element source with a link to the logo or properly imp
...
+
+.. code-block:: yaml
+
+    # config.yaml
+
+    sylius_ui:
+    events:
+        sylius.shop.layout.header.grid:
+            blocks:
+                logo: 'bundles/SyliusShopBundle/Layout/Header/_myLogo.html.twig'
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/shop-customizations.rst:44]:
```suggestion
                logo: 'templates/shop/Layout/Header/_logo.html.twig'
```

File: docs/getting-started-with-sylius/shop-customizations.rst (hunk 1 above)
Comment (arti0090) [reply docs/getting-started-with-sylius/shop-customizations.rst:33]: @lchrusciel Your proposal does not seems to work.
Also I am using example from above (which is exact copy of line 28), so I don't know if this change at least here would be correct 🤔 

image

File: docs/getting-started-with-sylius/shop-customizations.rst [hunk 3]
```diff
@@ -30,7 +30,24 @@ Next, replace the ``img`` element source with a link to the logo or properly imp
 `Symfony assets documentation <https://symfony.com/doc/current/best_practices/web-assets.html>`_ for more info).
 
 The other way to achieve this is to modify the configuration of the ``sylius.shop.layout.header.grid`` template event.
-Learn more about it in :doc:`this doc</customization/template>`.
+Using the same logo file as from example above ``templates/bundles/SyliusShopBundle/Layout/Header/_logo.html.twig``.
+Change the configuration file:
```
[the remark this answers is not in the document]
Comment (arti0090) [reply docs/getting-started-with-sylius/shop-customizations.rst:34]: There is something wrong with logic of this sentence here as _logo.html.twig is a file where html code is stored not a configuration

File: docs/getting-started-with-sylius/shop-customizations.rst (hunk 1 above)
Comment (lchrusciel) [reply docs/getting-started-with-sylius/shop-customizations.rst]: you are correct, line 28 of the `docs/getting-started-with-sylius/shop-customizations.rst` file should be adjusted as well, where we should replace the `templates/bundles/SyliusShopBundle/Layout/Header/_logo.html.twig` file with `templates/shop/Layout/Header/_logo.html.twig`

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @arti0090! :tada:

**Pull request merged**